### ISSUE TITLE
keyboard in the middle of the screen

### ISSUE THREAD

**Issue by rafael-h-ferreira** [ionic-team/cordova-plugin-ionic-keyboard#99]
When clicking inside a input, the keyboard shows up but goes to high up in the screen and with blank keys. Any thoughts?
Thanks

![sreenshot_keyboard_above](https://user-images.githubusercontent.com/2298363/62784649-df9e0480-bab6-11e9-98fe-fc93b5f94a40.PNG)

Ionic:

   Ionic CLI                     : 5.2.3 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.5.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   Cordova CLI       : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms : ios 4.5.5
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.1.1, (and 9 other plugins)

Utility:

   cordova-res : 0.6.0 
   native-run  : 0.2.8 

System:

   ios-deploy : 1.9.4
   ios-sim    : 8.0.1
   NodeJS     : v10.16.0 (/usr/local/bin/node)
   npm        : 6.10.2
   OS         : macOS High Sierra
   Xcode      : Xcode 10.1 Build version 10B61

**Comment by jcesarmobile** [comment]
I've never seen something like this, can you provide a sample app?

Are you using a default keyboard or did you install a custom one?

**Comment by ctmartinez1992** [comment]
Hey, we've solved this issue in the meantime.

I can't say accurately what was that solved it, but I believe that disabling the view resizing when opening the keyboard solved all the issues. However, I'm afraid this doesn't work with view resizing on.

The only way we got it to work with view resizing in ios 12, was by compiling it with xcode 9. If I wanted to compile it in xcode 10, I had to compile it to ios 11.x, but is not optimal at all since most iphones are probably on version 12 by now.

Btw, this issue was present in this plugin, in the previous plugin "cordova plugin keyboard" by cjpearson, and even on this paid [version](https://github.com/EddyVerbruggen/cordova-plugin-native-keyboard). I even tried many forks that had a similar issue with resizing views and none of it worked.
